cmd/preflight/cmd/root.go

```diff
@@ -104,19 +114,7 @@ func preRunConfig(cmd *cobra.Command, args []string) {
 		l.SetLevel(ll)
 	}
 
-	// if we are in the offline flow redirect log file to exist in the directory where all other artifact exist
-	if viper.GetBool("offline") {
-		artifacts := viper.GetString("artifacts")
-
-		// ignoring error since OpenFile will error and we'll still have the multiwriter from above
-		_ = os.Mkdir(artifacts, 0o777)
-
-		artifactsLogFile, err := os.OpenFile(filepath.Join(artifacts, logname), os.O_CREATE|os.O_WRONLY|os.O_TRUNC, 0o600)
-		if err == nil {
-			mw := io.MultiWriter(os.Stderr, logFile, artifactsLogFile)
-			l.SetOutput(mw)
-		}
```

Comment:
This code should stay, the intent is to have the log file in the `tar` and at the default location.